Tests/NIOWebSocketTests/WebSocketMaskingKeyTests.swift

```diff
@@ -31,4 +31,11 @@ final class WebSocketMaskingKeyTests: XCTestCase {
             WebSocketMaskingKey.random(using: &generator) != [0, 0, 0, 0]
         }, "at least 1 of 1000 random masking keys should not be all zeros")
     }
+    
+    func testRandomMaskingKeyWithDefaultGenerator() {
+        let key1 = WebSocketMaskingKey.random()
+        let key2 = WebSocketMaskingKey.random()
+        
+        XCTAssertNotEqual(key1, key2)
```

Comment:
This test has a non-zero chance of failing: a random sequence is not forced to avoid repeating values. Can you replace this with a variant of the test directly above? I'm fine with you calling `random` deterministically at least twice to produce your test case for the Swift repo.

Reply:
Done, good call 👍